src/TestReports.jl

```diff
@@ -31,9 +31,23 @@ end
 
 export ReportingTestSet, any_problems, report, recordproperty
 
+const TESTREPORTS_VERSION = let # Copied from Documenter.jl
+    project = joinpath(dirname(dirname(pathof(TestReports))), "Project.toml")
```

Comment:
Technically this could balso be `JuliaProject.toml`
Idk if that exists in wild though

Reply:
As it's just loading from `TestReports`, I think it's probably okay to be hardcoded to `Project.toml` as we're not going to change it to a `JuliaProject.toml`?